.github/workflows/node-ci.yml

```diff
@@ -11,12 +11,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out repository
-        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # renovate: tag=v2
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2
 
       - name: Set up node
         uses: actions/setup-node@04c56d2f954f1e4c69436aa54cfef261a018f458 # renovate: tag=v2
         with:
-          node-version: '14.18.1'
+          node-version: '14.18.2'
```

Comment:
unclear how/why these changes are in this branch and not main, and how I got them. will review tomorrow 😩

Reply:
it's showing a number of files in changed that are just in main... maybe it will resolve itself tomorrow.

Reply:
⭐ resolved ⭐